### DISCUSSION TITLE
net::ERR_ABORTED 404 when cookie is disabled

### DISCUSSION THREAD

**Discussion by jamal299** [vercel/next.js#16363] [Help]
# Bug report

## Describe the bug
 We are deploying our next project using npm run build and npm run start. It works fine on all browsers. When all cookies are disabled the browser returns net::ERR_ABORTED 404. for js and css files (Slug.js, _buildManifest.js and chunk.css) some files return 200 and some return 404.

## To Reproduce
npm run build
npm run start

Steps to reproduce the behavior, please provide code snippets or a repository:

disable all cookies in chrome, npm run build and then npm run start...

## Expected behavior

We expect the application to load properly on a browser where cookie is disabled as it is loading on a browser where cookies are enabled.

## System information

- OS: [e.g. macOS, Windows]
- Browser (if applies)   all browsers
- Version of Next.js: 9.4.4
- Version of Node.js: 13.5.0

## Additional context

<img width="711" alt="Screen Shot 2020-08-19 at 8 53 30 PM" src="https://user-images.githubusercontent.com/26132005/90678643-92817080-e27c-11ea-8c82-ec71d7d1fc95.png">

**Comment by timneutkens** [comment]
Please add a complete reproduction.
